index.html

```diff
@@ -35,5 +35,156 @@ <h2>I'm a Software Developer</h2>
             </ul>
         </div>        
     </section>
+    <section class="works">
+        <div class="card-1">
+            <span>My Recent works</span>
+        </div>
+        <div class="card-2">
+            <img src="/images/gym-image.png" alt="Placeholder">
+            <p id="story">Multi-Post Stories</p>
+            <p>
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+                when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="link-1">
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">RUBY</a></li>
+            </ul>
+            <a class="see-project button-1" href="#">See project</a>
```

Comment:
- [ ] Kindy add the button interactions to your button styling: Active, Hover, and Pressed (see design) so as to improve your website accessibility 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/227714194-e8ae32d4-895e-4aa6-8503-a1b49956d631.png)